### ISSUE TITLE
3 dot menu disappears when item is clicked on

### ISSUE THREAD

**Issue by ArjunSingh99** [martinpl/tray-icons-reloaded#32]
When the 3 dot menu appears, and an app is right clicked on, the menu disappears. This behaviour is not coherent with the default shell, and doesn't look very good either. Please consider adding it as an option or changing it for good.

Example:
![image](https://user-images.githubusercontent.com/53963483/114919299-b31ebc00-9e45-11eb-9c49-d3c4fe8ef54e.png)

Unfortunately, I'm unable to take a screenshot of what it looks like after right click because of #31, so here's a camera capture instead:

https://user-images.githubusercontent.com/53963483/114919977-8fa84100-9e46-11eb-8c69-8967258b4fce.mp4

**Comment by cesar19004** [comment]
Yeah, same problem, the menu disappears when an app is right clicked
